.env.example

```
@@ -232,3 +235,30 @@ SECRETS_CAP_ENFORCEMENT=
+#   openssl rand -hex 16
+# The AWS_* and LITESTREAM_* credentials are these same values when the
+# stack's MinIO is the object store.
+MINIO_ROOT_USER=minioadmin
+MINIO_ROOT_PASSWORD=
+# Where Litestream replicates DATA_DIR/global.db, read by the sidecar and by
+# the app image's restore-on-empty entrypoint. The compose stack requires a
+# bucket name; only the image run on its own treats an empty value as "no
+# replica". LITESTREAM_ENDPOINT is empty for AWS S3.
+LITESTREAM_BUCKET=backups
+LITESTREAM_ENDPOINT=http://minio:9000
+LITESTREAM_ACCESS_KEY_ID=minioadmin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LITESTREAM_ACCESS_KEY_ID key should go before the LITESTREAM_BUCKET key

(UnorderedKey)